shop/src/main/resources/db/migration/V1_1_0__create_location_schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE Location (
```

Comment:
Have a look over the [SQL Style Guide](https://www.sqlstyle.guide/). Use snake case for table and field names (e.g., `location`, `address_street_address`). 

You could drop the `address` prefix from the name of the fields (e.g., `address_country` => `country`). Their meaning is clear without `address` in the name. This will also remove the need for `AttributeOverrides` in `Location.java`.

Think whether it makes sense to have `NOT NULL` on other fields. For exemple, do we want to allow `name` to be null?

Make the adjustments to all tables.